### ISSUE TITLE
Sorting options

### ISSUE THREAD

**Issue by toms0815** [IdreesInc/Waypoint#38]
Thanks for this great plugin!
It would be very useful to have an option to change the way notes are listed, e.g. to sort them by creation date.

**Comment by pcause** [comment]
agree, great plugin and thanks.  +1 for sorting and having the option to change in the page not just options.

**Comment by marconoris** [comment]
+1 for me 👍

**Comment by kimaldis** [comment]
+1 also from me, with the addition that I can have folders shown at the top of the list, then notes. Make the layout of notes and folders clearer. Also, this would echo the way the structure shows in the file explorer.

Extremely useful plugin.
